src/main/java/com/google/api/codegen/util/java/JavaTypeTable.java

```diff
@@ -14,6 +14,8 @@
  */
 package com.google.api.codegen.util.java;
 
+import static com.google.api.codegen.util.java.JavaTypeTable.JavaLangResolution.ESCAPE_JAVA_LANG_CLASH;
```

Comment:
JavaLangResolution is in JavaTypeTable. The static import is there so the enum doesn't have to be qualified when it's used, even though the enum is defined in this class.